UnityProject/Assets/Sprites/0x72_DungeonTilesetII_v1.1.png.meta

```diff
@@ -519,21 +519,21 @@ TextureImporter:
   platformSettings:
   - serializedVersion: 2
     buildTarget: DefaultTexturePlatform
-    maxTextureSize: 2048
+    maxTextureSize: 8192
     resizeAlgorithm: 0
     textureFormat: -1
-    textureCompression: 0
+    textureCompression: 2
```

Comment:
Make sure compression is still none and filter to point. I wasn't expecting this file to change